src/Microsoft.AspNetCore.Http.Connections/ConnectionsRouteBuilder.cs

```diff
@@ -10,20 +10,39 @@
 
 namespace Microsoft.AspNetCore.Http.Connections
 {
+    /// <summary>
+    /// Maps routes to the HTTP connection dispatcher.
```

Comment:
"Maps routes to ASP.NET Core Connection Handlers" (granted, there's a version that just takes a delegate, but we could argue that's still a "connection handler" ;)